src/client/fdsn.py

```diff
@@ -22,7 +22,7 @@
     from urllib.error import HTTPError
 
 from pyrocko import util
-
+from .. import config
```

Comment:
better use `from pyrocko import config` in this case... explicit than implicit

Reply:
agreed